test/syntax.jl

```diff
@@ -685,7 +685,6 @@ function count_meta_loc(exprs)
         end
         @test push_count >= pop_count
     end
-    @test push_count == pop_count
```

Comment:
looks like we weren't counting these properly anymore either (`pop_loc` takes an argument)